### PR TITLE
gitlab-ce to 9.4.5 and gitlab-runner to 9.4.2 added as new version

### PR DESCRIPTION
**Pull request by fluxibox**
We updated our running gitlab-ce to 9.4.5 and gitlab-runner to 9.4.2. Push to private registry works perfectly now!

### CODE REVIEW

File: templates/gitlab-multi-runner/2/docker-compose.yml.tpl [hunk 1]
```diff
@@ -0,0 +1,47 @@
+version: '2'
+
+services:
+
+  gitlab-runner-config:
+    image: gitlab/gitlab-runner:alpine-v9.4.1
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/2/docker-compose.yml.tpl]: Version 9.4.2??
```
-    image: gitlab/gitlab-runner:alpine-v9.4.1
+    image: gitlab/gitlab-runner:alpine-v9.4.2
```

File: templates/gitlab-multi-runner/2/docker-compose.yml.tpl [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    - --docker-image
+    - docker:latest
+    - --docker-volumes
+    - /var/run/docker.sock:/var/run/docker.sock
+    - --docker-privileged
+    labels:
+      io.rancher.container.start_once: 'true'
+
+  gitlab-runner:
+    image: gitlab/gitlab-runner:alpine-v9.4.1
```
Comment (rawmind0) [inline templates/gitlab-multi-runner/2/docker-compose.yml.tpl]: Version 9.4.2??
```
-    image: gitlab/gitlab-runner:alpine-v9.4.1
+    image: gitlab/gitlab-runner:alpine-v9.4.2
```

**Comment by rawmind0** [comment]
Hi @fluxibox ...thanks for the PR....

I think gitlab-runner image version is wrong. Could you please take a look??

**Comment by fluxibox** [comment]
gitlab-runner image version is wrong. Could you please take a look?? yes

Sorry it's now changed from 9.4.1 to  9.4.2 I checked the actual running image after updateing in rancher. 

Runner can connect to gitlab and build a projects successfuly.

**Comment by rawmind0** [comment]
Thanks @fluxibox ...

LGTM..

**Pull request merged**